### PR TITLE
Week1

### PR DESCRIPTION
**Pull request by PotzarellaMozarella**
Updated the Assignments for Week 1:
-How browsers work
-Static website page

### CODE REVIEW

File: Week-1/index.html [hunk 1]
```diff
@@ -0,0 +1,114 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
```
Comment (imranmd) [inline Week-1/index.html:7]: you can add any other meta tags that you have learnt

File: Week-1/index.html [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+                        <div class="playlist-control">
+                            <a href="#"><i class="fa-solid fa-play control"></i>Play all</a>
+                            <a href="#"><i class="fa-solid fa-plus control"></i>Add to</a>
+                            <a href="#"><i class="fa-solid fa-minus control"></i>More</a>
+                        </div>
+                    </div>
+                    
+                </div>
+            </div>
+            //list of songs section
```
Comment (imranmd) [inline Week-1/index.html:57]: A better comment format:

File: Week-1/style.css [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+    {
+        padding: 2rem;
+
+    }
+.control {
+    padding-right: 0.4rem;
+}
+
+/* playlist table styling */
+table {
```
Comment (imranmd) [inline Week-1/style.css:74]: adding styles to the tags like h1 or td , th or table will apply same styles across the project.

A best practice is to create a class or apply styles to ids

File: Week-1/style.css [hunk 4]
```diff
@@ -0,0 +1,136 @@
...
+    padding: 0.2rem;
+    
+}
+
+td {
+    padding: 1rem 0.8rem 1rem 0;
+  }
+  
+tr:nth-child(even) {
+    background-color: rgba(241, 241, 241, 0.9);
```
Comment (imranmd) [inline Week-1/style.css:85]: color codes are better than RGB format

Comment (imranmd) [reply Week-1/style.css:85]: Try to make it responsive using media queries

File: Week-1/Readme.md [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+Used to parse and execute JavaScript code. Add the interactivity to the page. 
+- **Data storage:** 
+This is a persistence layer. The browser may need to save all sorts of data locally, such as cookies. Browsers also support storage mechanisms such as localStorage, IndexedDB, WebSQL and FileSystem.
+Markdown is a lightweight markup language based on the formatting conventions
+that people naturally use in email.
+
+#### _How does the rendering process work?_
+The below diagram explains the overall rendering engine flow:
+
+![Rendering engine flow](Images/img4.png)
```
Comment (imranmd) [inline Week-1/Readme.md:41]: This is a good one

File: Week-1/index.html (hunk 2 above)
Comment (PotzarellaMozarella) [reply Week-1/index.html:58]: what does this mean, are you referring to the indentation?

File: Week-1/index.html (hunk 1 above)
Comment (PotzarellaMozarella) [reply Week-1/index.html]: added

File: Week-1/index.html (hunk 2 above)
Comment (PotzarellaMozarella) [reply Week-1/index.html]: done

File: Week-1/style.css (hunk 3 above)
Comment (PotzarellaMozarella) [reply Week-1/style.css]: done

File: Week-1/style.css (hunk 4 above)
Comment (PotzarellaMozarella) [reply Week-1/style.css]: done

**Pull request merged**